### ISSUE TITLE
p5.js library links in /examples 

### ISSUE THREAD

**Issue by therewasaguy** [processing/p5.js-sound#484]
With these

https://github.com/processing/p5.js-sound/pull/483
https://github.com/processing/p5.js-sound/issues/480

we're no longer using outdated p5.js library in our examples, which is great.

I wonder what we should do about the /examples which now have broken links in their index.html like these https://github.com/processing/p5.js-sound/blob/master/examples/Compressor/index.html#L4-L6

If there were a way to get "latest" from the CDN, that'd be ideal...

Or, could we download / update the latest when running `npm install`?

@montoyamoraga curious if you have any ideas here?

**Comment by montoyamoraga** [comment]
@therewasaguy i agree, i like it that now we don't have that manual step of updating p5.js library, and having that issue of having older versions, yay.

i think there is no way of pointing to the latest version of a library using CDN, and i found this stackoverflow and i see the point of not having one :)
https://stackoverflow.com/questions/14801334/getting-the-latest-version-from-cdnjs

i rather do your option of using ```npm install```, because if we replaced the examples to rely from CDN we would still need to update manually, but i don't know how to do that, do you @therewasaguy? :)

**Comment by therewasaguy** [comment]
Good research!

So then the examples could deep link within the node_modules directory...maybe not ideal but it could work.

Looks like we'd need to update our github pages to get this to work for http://processing.github.io/p5.js-sound/  https://stackoverflow.com/a/43487798/2994108

**Comment by endurance21** [comment]
@therewasaguy  i hope we are done with this issue ?
